Cargo.toml

```diff
@@ -7,3 +7,6 @@ members = [
     "proving_system/plonk",
     "gpu",
 ]
+
+[workspace.dependencies]
```

Comment:
Shouldn't this be something like `[workspace.dev-dependencies]`?

Reply:
No. I would have hoped so, but there's only `dependencies` at the workspace level. But don't worry, those only get imported and built when called, so as long as we only add them to crates in the `dev-dependencies` section they won't end up in the release builds.